husky_bringup/launch/bumblebee/bumblebee.launch

```diff
@@ -0,0 +1,38 @@
+<launch>
```

Comment:
Chain to the full bumblebee launch from a small launch if necessary, but no need to reproduce ALL of it here.